### DISCUSSION TITLE
Upcoming ZOSPy updates

### DISCUSSION THREAD

**Discussion by jwmbeenakker** [MREYE-LUMC/ZOSPy#51] [Announcements]
We want to give you a heads-up for two ZOSPy updates we are planning to release in the coming months:

We aim to release **version 1.2** in the beginning of January, which will include a number of small improvements and new features such as:
- #20 
- #34 and #41
- #47  (when the functionality on the OpticStudio forum actually works)
- [`doctest`](https://docs.python.org/3/library/doctest.html) to test the examples of the doc strings
- some new analyses

Afterwards, we will work on #15, which will be a bigger release and might introduce breaking changes.

If you have any pull requests in the pipeline, please let us know, so we can schedule accordingly.

**Comment by Omnistic** [comment]
This is amazing, thank you @jwmbeenakker and everyone who contributed to ZOSPy. I cannot program as well as you but please let me know if there's anything I can do to help ZOSPy grow. Take care, David.
